### ISSUE TITLE
Sourcemaps

### ISSUE THREAD

**Issue by pelmered** [sanatorium/wp-vue-boilerplate#25]
I've been trying to get sourcemaps to be generated in this webpack config but I can't get it to work. I'm not really used to the loaders and plugins you use. How would you do to enable sourcemaps? I prefer a separate file, but inline is also fine. 

Sourcemaps is probably something that the theme should have out of the box.

**Comment by matejpozdravuje** [comment]
Thanks for your point. We merged branches 0.2.0 and master and changed webpack configuration, so it has now sourcemaps out of the box. If you need you can specify a level of source mapping in ```/config/index.js```  at line 12 ```devtool: '#source-map'```. Available styles of source mapping can be found here https://webpack.js.org/configuration/devtool/
By default sourcemaps are generated to ```/dist/static/js``` and ```/dist/static/css```

**Comment by pelmered** [comment]
@MattLuk112 Thanks!
I tried to upgrade my project with the new build system(I replaced the `build` and `config` folders and made some other small adjustments, but it doesn't build. I don't get any error messages neither, just this: https://www.dropbox.com/s/bu4ha3wk25khuy8/Screenshot%202017-10-13%2016.18.40.png?dl=0

The dev server just give me `Cannot GET /` . Any ideas what I should do?

I think the new build system is somewhat complicated which makes it hard to follow and debug. Could it be simplified a bit?
